src/redux/configureStore.js

```diff
@@ -0,0 +1,21 @@
+import { createStore, applyMiddleware, combineReducers } from 'redux';
+import thunk from 'redux-thunk';
+import logger from 'redux-logger';
+import rocketsReducer, { fetchRocketsThunk } from './rockets/rockets';
+import reservedRocketsReducer, {
+  fetchReservedRocketsThunk,
+} from './profile/profile';
+import missionsReducer, { fetchMissionsThunk } from './missions/missions';
+
+const rootReducer = combineReducers({
+  rocketsReducer,
+  reservedRocketsReducer,
+  missionsReducer,
+});
```

Comment:
- [ ] I'm getting this error message when I run your app on my local machine 👇🏻 Kindly make sure to fix this 👍🏻 

![18](https://user-images.githubusercontent.com/79058364/193144620-70c9442a-a3ff-4ca1-946c-cb1120f07e76.png)

Reply:
Use the command `npm install` to install redux and other necessary packages for the project before `npm start`